proxy-robots-module/src/main/java/orwell/proxy/robot/ColourSensor.java

```diff
@@ -85,4 +81,18 @@ public void clear() {
     public void accept(final IRobotElementVisitor visitor) {
         visitor.visit(this);
```

Comment:
Some visitors cannot be null while some others can, I find this quite confusing.